test/test_block_token.py

```diff
@@ -377,6 +377,22 @@ def test_footnotes_may_not_have_too_much_leading_space(self):
         self.assertIsInstance(token.children[2], block_token.BlockCode)
         self.assertEqual(token.children[2].children[0].content, "[i-am-block-too]: /foo\n")
 
+    def test_footnotes_angle_bracketed_link_with_space(self):
+        lines = ['[Foo bar]:\n',
+                 '<my url>\n',
+                 '\'title\'\n',
+                 '\n',
+                 '[Foo bar]\n']
+        token = block_token.Document(lines)
+        self.assertEqual(token.footnotes, {core_tokens.normalize_label("Foo bar"): ("my url", "title")})
+
+    def test_footnotes_title_separated_from_link_destination(self):
```

Comment:
This is an extra test added & taken from Example 201 in spec, right? As in the case above, I would suggest to simplify it, like this:

```python
    def test_no_footnote_when_title_right_after_link_destination(self):
        lines = ['[foo]: <bar>(baz)\n']
        token = block_token.Document(lines)
        self.assertEqual(token.footnotes, {})
```

Reply:
ok. added a positive test, too.

Reply:
so now it is `test_parse_title_must_be_separated_from_link_destination()` - OK